File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/ExportSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface ExportSearchModel extends BaseSearchModel {
    fromDate: Date | null;
    toDate: Date | null;
    clientId: number | null;
    clientCode: string;
    branchId: number | null;
    branchCode: string;
    storeId: number | null;
    storeCode: string;
    vendorId: number | null;
    vendorCode: string;
    itemId: number | null;
    itemCode: string;

    representativeId:number |null;
    representativeCode:string |null;

}

File: SofiForce.Models/Models/SearchModels/ExportSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ExportSearchModel : BaseSearchModel
    {
		public DateTime? FromDate { get; set; }
		public DateTime? ToDate { get; set; }

		public Int32? ClientId { get; set; }

		public Int32? BranchId { get; set; }

		public Int32? StoreId { get; set; }


		public Int32? VendorId { get; set; }


		public Int32? RepresentativeId { get; set; }
		

		public Int32? ItemId { get; set; }

		


	}
}